src/pages/AppDetail.svelte

```diff
@@ -1,17 +1,14 @@
 <script>
+  import { fetchJSON } from "../state/api";
+
   export let params;
   const URL = `data/${params.app}/index.json`;
-  let app;
-  fetch(URL)
-    .then((r) => r.json())
-    .then((ret) => {
-      app = ret;
-    });
+  let app = fetchJSON(URL);
```

Comment:
I've dealt with async sections like this using the `onMount` lifecycle function, but the `await` keyword in the document works too. Just for reference, it would look something like this:

```javascript
import { onMount } from "svelte";

...

onMount(async () => {
  let app = await fetchJSON(URL);
});
```